bot/bot.py

```diff
@@ -62,6 +70,15 @@ async def on_thread_join(self, thread: Thread) -> None:
         with suppress(Forbidden):
             await thread.join()
 
+    async def start(self, *args, **kwargs) -> None:
+        """Async wrapper to load all the extensions."""
+        from bot.utils.extensions import walk_extensions
```

Comment:
why is this imported here? is it possible to import it at the top of the file?

Reply:
Circular imports.